src/core/config.py

```diff
@@ -22,6 +22,7 @@ def __init__(self):
         
         # Model settings - BIG and SMALL models
         self.big_model = os.environ.get("BIG_MODEL", "gpt-4o")
+        self.middle_model = os.environ.get("MIDDLE_MODEL", "gpt-4o")
```

Comment:
Can we add some forward compatibility methods to avoid situations where it cannot run after a ⁠git pull

Reply:
Could you provide a failure example?

Reply:
If the user has not configured MIDDLE_MODEL, can we use BiG_MODEL as the default value? One scenario is that the user configured Big/Small model, but the newly added Middle model uses the default 4o, causing the service to error😢